internal/command/postgres/postgres.go

```diff
@@ -115,6 +116,22 @@ func hasRequiredVersionOnMachines(appName string, machines []*fly.Machine, clust
 	return nil
 }
 
+func hasRequiredFlexVersionOnMachines(appName string, machines []*fly.Machine, flexVersion string) error {
+	if len(machines) == 0 {
+		return fmt.Errorf("no machines provided")
+	}
+
+	if !IsFlex(machines[0]) {
+		return fmt.Errorf("not a Flex cluster")
+	}
+
+	err := hasRequiredVersionOnMachines(appName, machines, "", flexVersion, "")
+	if strings.Contains(err.Error(), "Malformed version") {
+		return fmt.Errorf("This image is not compatible with this feature. Please attempt an update with `fly image update -a %s`", appName)
```

Comment:
I don't think an image update would not address the malformed version error.  You'd typically hit a malformed version error when a non-compatible App tries to use the feature, or someone's running a forked version of Flex.

Reply:
Fair point, I can remove the update CTA.